packages/easy-coding-standard/packages/SniffRunner/Application/SniffFileProcessor.php

```diff
@@ -110,7 +111,7 @@ public function processFile(SmartFileInfo $smartFileInfo, Configuration $configu
     public function processFileToString(SmartFileInfo $smartFileInfo): string
     {
         $file = $this->fileFactory->createFromFileInfo($smartFileInfo);
-        $this->fixFile($file, $this->fixer, $smartFileInfo, $this->tokenListeners);
+        $this->fixFile($file, $this->fixer, [], $smartFileInfo, $this->tokenListeners);
```

Comment:
I've missed this. In general, the new parameter should be always in the end. 

Could you move this one too?

Reply:
It's a private method on a final class, but I get your point, moved